static/js/pages/repo-view.js

```diff
@@ -77,8 +77,8 @@
         if (resp.has_additional) {
           loadPaginatedRepositoryTags(page + 1);
         } else {
-	  $scope.viewScope.tagsLoading = false;
-	}
+          $scope.viewScope.tagsLoading = false;
+        }
```

Comment:
nit: extra spaces

Reply:
not sure I see it, looks correctly formatted in vscode 🙃

Reply:
Sometimes just deleting all the extra spaces and adding a tab works. Super minor

Reply:
This diff replaces tab with spaces. Looks correct to me.